app/src/main/java/com/codesoom/assignment/httphandler/TaskHttpHandler.java

```diff
@@ -0,0 +1,111 @@
+package com.codesoom.assignment.httphandler;
+
+import com.codesoom.assignment.model.Task;
+import com.codesoom.assignment.util.JsonObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+	static int taskId = 0;
+	private static final List<Task> tasks = new ArrayList<>();
+
+	@Override
+	public void handle(HttpExchange httpExchange) throws IOException {
+		String requestMethod = httpExchange.getRequestMethod();
+		String path = httpExchange.getRequestURI().getPath();
+
+		if ("GET".equals(requestMethod) && "/tasks".equals(path)) {
+			getAllTasks(httpExchange);
+		} else if ("GET".equals(requestMethod) && path.startsWith("/tasks/")) {
+			getTask(httpExchange);
+		} else if ("POST".equals(requestMethod)) {
+			createTask(httpExchange);
+		} else if ("PUT".equals(requestMethod)) {
+			putTask(httpExchange);
+		} else if ("DELETE".equals(requestMethod)) {
+			deleteTask(httpExchange);
+		}
+	}
+
+	private void deleteTask(HttpExchange httpExchange) throws IOException {
+		String path = httpExchange.getRequestURI().getPath();
+		int taskId = extractTaskIdFromPath(path);
+		Task task = findTask(taskId);
+
+		if (task != null) {
+			tasks.remove(task);
+		}
+		response(httpExchange, 200, null);
```

Comment:
```suggestion
		Task task = findTask(taskId);
		if (task != null) {
			tasks.remove(task);
		}

		response(httpExchange, 200, null);
```

관련이 있는 코드들끼리는 가까이 위치하고, 관련이 없는 코드들은 빈 줄로 구분해주는 것이 좋습니다. 우리가 글을 읽을 때 문단이 적절히 나눠져있다면, 새로운 내용이 시작된다는 것을 알고 읽잖아요. 코드도 마찬가지입니다. 그래서 비슷한 맥락끼리 같이 읽을 수 있도록 적절히 묶어주고, 관련이 없는 코드들은 적절히 구분해주는게 좋습니다.